test/dag.spec.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+
+const IPFSApi = require('../src')
+const f = require('./utils/factory')
+
+describe('.dag', function () {
+  this.timeout(50 * 1000) // slow CI
```

Comment:
I guess that was copy&pasted. Generally try to stick to the default timeouts and see if they are long enough,

Reply:
I tried with the default timeout but it was not enough.